README.md

```diff
@@ -339,12 +339,6 @@ Full Instructions:
   git clone https://github.com/<user>/universe.git /path/to/universe
   ```
 
-2. Ensure that the `jsonschema` command line tool is installed:
-
-  ```bash
-  sudo pip install jsonschema
-  ```
-
```

Comment:
We don't need this instruction anymore. The build script is installing the necessary requirements.